3rd-party/wpml.php

```diff
@@ -32,4 +32,51 @@ function grunion_contact_form_field_html_filter( $r, $field_label, $id ){
 	return $r;
 }
 
+/**
+ * Add current language as suffix to transient name.
+ *
+ * @param string $transient Transient name. Expected to not be SQL-escaped.
+ * @return string Translated transient name.
+ */
+add_filter( 'jetpack_sitemap_transient', 'jetpack_wpml_transient_language', 10, 1 );
+add_filter( 'jetpack_news_sitemap_transient', 'jetpack_wpml_transient_language', 10, 1 );
+add_filter( 'jetpack_sitemap_xsl_transient', 'jetpack_wpml_transient_language', 10, 1 ) ;
+
```

Comment:
Can you please move the inline doc that's written a few lines above to this location?